requirements/run.txt

```diff
@@ -0,0 +1,3 @@
+scipy
+numpy
+matplotlib
```

Comment:
the next probably thing is adding `diffpy.structure` once released to see if it works..

Reply:
@sbillinge 

While we have the core under`run.txt` and just create `run_ext.txt` for those extra dependencies?

Since our centralized GH CI workflow files need to be updated:

from:

```
      - name: Install ${{ inputs.project }} and requirements
        run: |
          conda install --file requirements/run.txt
```

to:

```
      - name: Install ${{ inputs.project }} and requirements
        run: |
          conda install --file requirements/run.txt
          conda install --file requirements/run-ext.xt
```

Reply:
Yes, good idea